iceoryx_posh/test/moduletests/test_popo_chunk_queue.cpp

```diff
@@ -144,56 +146,52 @@ TEST_P(ChunkQueue_test, ClearWithData)
     EXPECT_THAT(m_popper.empty(), Eq(true));
 }
 
-TEST_P(ChunkQueue_test, AttachSemaphore)
+TEST_P(ChunkQueue_test, AttachConditionVariableSignaler)
 {
-    auto semaphore = semaphorePool.createObjectWithCreationPattern<iox::posix::Semaphore::errorType_t>(0);
-    ASSERT_THAT(semaphore.has_error(), Eq(false));
+    ConditionVariableData condVar;
 
-    auto ret = m_popper.attachSemaphore(*semaphore);
-    EXPECT_FALSE(ret.has_error());
+    auto ret = m_popper.attachConditionVariableSignaler(&condVar);
+    EXPECT_TRUE(ret);
 
-    EXPECT_THAT(m_popper.isSemaphoreAttached(), Eq(true));
+    EXPECT_THAT(m_popper.isConditionVariableSignalerAttached(), Eq(true));
 }
 
 TEST_P(ChunkQueue_test, DISABLED_PushAndTriggersSemaphore)
```

Comment:
Rename test for condition variable. Is this test still required as it's disabled?

Reply:
Renamed the test. I don't know why @budrus disabled this test case. I suppose due to the missing lock in `attachConditonVariableSignaler` and `detachConditionVariableSignaler`.